frontend/packages/employee-frontend/src/components/reports/VoucherServiceProviders.tsx

```diff
@@ -0,0 +1,209 @@
+      value: year.toString(),
+      label: year.toString()
+    })
+  }
+  return yearOptions
+}
+
+function getFilename(
+  i18n: Translations,
+  year: number,
+  month: number,
+  careAreaName: string
+) {
+  const time = formatDate(new Date(year, month - 1, 1), 'yyyy-MM')
+  return `${time}-${careAreaName}.csv`.replace(/ /g, '_')
+}
+
+function VoucherServiceProviders() {
+  const { i18n } = useTranslation()
+  const [rows, setRows] = useState<Result<VoucherServiceProviderRow[]>>(
+    Success([])
+  )
+  const [areas, setAreas] = useState<CareArea[]>([])
+  const [filters, setFilters] = useState<VoucherServiceProvidersFilters>({
+    year: new Date().getFullYear(),
+    month: new Date().getMonth() + 1,
+    careAreaId: ''
+  })
+
+  useEffect(() => {
+    void getAreas().then((res) => isSuccess(res) && setAreas(res.data))
+  }, [])
+
+  useEffect(() => {
+    if (filters.careAreaId == '') return
+
+    setRows(Loading())
+    void getVoucherServiceProvidersReport(filters).then(setRows)
+  }, [filters])
+
+  const months = monthOptions()
+  const years = yearOptions()
+
+  return (
+    <Container>
+      <ReturnButton />
+      <ContentArea opaque>
+        <Title size={1}>{i18n.reports.voucherServiceProviders.title}</Title>
+        <FilterRow>
+          <FilterLabel>{i18n.reports.common.period}</FilterLabel>
+          <FlexRow>
+            <Wrapper>
+              <ReactSelect
+                options={months}
+                value={months.find(
+                  (opt) => opt.value === filters.month.toString()
+                )}
+                onChange={(value) => {
+                  if (value && 'value' in value) {
+                    const month = parseInt(value.value)
+                    setFilters({ ...filters, month })
+                  }
+                }}
+                styles={reactSelectStyles}
+              />
+            </Wrapper>
+            <Wrapper>
+              <ReactSelect
+                options={years}
+                value={years.find(
+                  (opt) => opt.value === filters.year.toString()
+                )}
+                onChange={(value) => {
+                  if (value && 'value' in value) {
+                    const year = parseInt(value.value)
+                    setFilters({ ...filters, year })
+                  }
+                }}
+                styles={reactSelectStyles}
+              />
+            </Wrapper>
+          </FlexRow>
+        </FilterRow>
+        <FilterRow>
+          <FilterLabel>{i18n.reports.common.careAreaName}</FilterLabel>
+          <Wrapper>
+            <ReactSelect
+              options={[
+                ...areas.map((area) => ({ value: area.id, label: area.name }))
+              ]}
+              onChange={(value) => {
+                if (value && 'value' in value) {
+                  setFilters({ ...filters, careAreaId: value.value })
+                }
+              }}
+              styles={reactSelectStyles}
+            />
```

Comment:
You should probably the `placeholder` prop since the default `Select...` is not ideal.